less/serialConsole.less

```diff
@@ -0,0 +1,78 @@
+/**
+  SerialConsole styles
+ */
+
+.console-ct > .terminal {
+  color: @color-pf-white;
+  text-align: left;
+  outline: medium none;
+  background-color: black;
+  border: 1px solid @color-pf-black;
+  padding: 10px;
+}
+
+.console-ct {
+  font-family: @font-family-monospace;
+  margin-top: 0;
+  margin-bottom: 0;
+  font-size: 10px;
+  text-align: center;
+  line-height: normal;
+}
+
+@media (min-width: 568px) {
```

Comment:
Would it still work if we used one of the bootstrap variables `@screen-sm` or `@screen-xs` or is there a reason for using 568?

Reply:
`@screen-tablet` used instead due to deprecation of the others. Just verified, still works well.